AirCasting/ABConnector/SDSyncController.swift

```diff
@@ -32,7 +32,7 @@ class SDSyncController: ObservableObject {
         self.sessionSynchronizer = sessionSynchronizer
     }
     
-    func syncFromAirbeam(_ airbeamConnection: CBPeripheral, progress: @escaping (SDCardSyncProgess) -> Void, completion: @escaping (Bool) -> Void) {
+    func syncFromAirbeam(_ airbeamConnection: CBPeripheral, progress: @escaping (SDCardSyncProgess) -> Void, finalizing: @escaping () -> Void, completion: @escaping (Bool) -> Void) {
```

Comment:
I think maybe switching from SDCardSyncProgress to something like:
```Swift
enum SDCardSyncStatus {
    case inProgress(SDCardSyncProgress)
    case finalizing
}
```
and having just a `progress` and `completion` blocks will be better? It looks a bit odd, I didn't quite get what is a difference between finalizing and completion at first